packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -100,6 +100,18 @@ class _DatePickerLayoutDelegate extends MultiChildLayoutDelegate {
   }
 }
 
+/// Retrieves cached widths of columns in the input state.
+@visibleForTesting
+Map<int, double> debugGetCupertinoDatePickerStateCache(State<CupertinoDatePicker> state) {
```

Comment:
This is awkward code to add to the framework. Can we find another way to test this?